### ISSUE TITLE
let bot see users outside of the current channel context

### ISSUE THREAD

**Issue by sjstein** [sjstein/R8DIUM#100]
Currently, the bot appears to be only able to bring up users with the @ tag if those users have access to the channel the bot is being invoked in.

Other bots (for example, YAG) do not suffer from this limitation. 

Find out how this is done and add that to r8dium.

**Comment by sjstein** [comment]
This is a matter of changing the variable type decorator in the function declarations.

Here is a "fixed" version of the change_roll command:

```
@client.tree.command(name='change_roll',
                         description=f'Set role <role_str> to user @id')
    @app_commands.describe(discord_id='@id', role='role name')
    async def change_roll(interaction: discord.Interaction, member: discord.Member, role: str):
        if CH_LOG != 'none':
            log_channel = discord.utils.get(interaction.guild.channels, name=CH_LOG)  # return channel id from name
            await log_channel.send(log_message(interaction))
        response = msgHandler.add_role(member.id, role, ldb)
        await interaction.response.send_message(response, ephemeral=True)  # noqa
```

Relevant changes are the change_roll(...member: discord.Member) and the use of that member attribute "id" in the msgHandler.add_role function call. Note that the member.id is an int, so the scrubbing off of the <@ > tags is not necessary.

**Comment by sjstein** [comment]
this was closed via merge #101
